### PR TITLE
feat: add entity migration cli

### PR DESCRIPTION
**Pull request by joaoandre-avaiga**

### CODE REVIEW

File: tests/data_sample/entities.json
Comment (jrobinAV) [inline tests/data_sample/entities.json:1]: Does it make sense to rename the file `entities_2.4.json`?

File: tests/data_sample/expected_entities.json
Comment (jrobinAV) [inline tests/data_sample/expected_entities.json:1]: Does it make sense to rename the file `entities_3.0.json` ?

**Review by jrobinAV — DISMISSED** [review]

**Review by trgiangdo — DISMISSED** [review]
I believe the test coverage is too low because we are not testing the `migrate` CLI directly. However, the current state of the PR is looking great. We can improve the coverage later if you guys don't mind.

**Comment by jrobinAV** [comment]
> I believe the test coverage is too low because we are not testing the `migrate` CLI directly. However, the current state of the PR is looking great. We can improve the coverage later if you guys don't mind.

Sounds good to me. I created a ticket for that:
https://app.zenhub.com/workspaces/taipy-613ef328affde6000ff77fb3/issues/gh/avaiga/taipy-core/776

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**